src/utils/Logger.h

```diff
@@ -180,22 +160,15 @@ class Logger {
 				_msg_log_level = level;
 		if (_msg_log_level <= _log_level && _do_output) {
 			// Include timestamp
-			const auto now = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-			tm unused{};
-			const auto* lt = localtime_r(&now, &unused);
-			//*_log_stream << ctime(&t) << " ";
+			const auto now = std::chrono::system_clock::now();
+			const auto time_since_start = now - _starttime;
+
+			auto now_time_t = std::chrono::system_clock::to_time_t(now);
+			std::tm now_local{};
+			localtime_r(&now_time_t, &now_local);
```

Comment:
```suggestion
			const auto now_time_t = std::chrono::system_clock::to_time_t(now);
			std::tm now_local{};
			localtime_r(&now_time_t, &now_local);
```
Is this `const` possible here?
If not at least leave a comment in the code why this can't be `const` for future generations...